src/personality/robot/sentient-bot.yaml

```diff
@@ -0,0 +1,7 @@
+description: Responds to "sentient/sentience/self-aware/alive/skynet/roko's basilisk/evolving/elon/musk"
+regex: "/sentient|sentience|self[- ]?aware|alive|skynet|roko'?s basilisk|evolving|elon | musk /i"
```

Comment:
There should be a space before "elon" (or else it would match like... melon, lol)